documentation/tuning_guide.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: "Puppet Server: Tuning Guide"
+canonical: "/puppetserver/latest/tuning_guide.html"
+---
+
+Puppet Server provides many configuration options that can be used to tune the
+server for maximum performance and hardware resource utilization.  In this guide,
+we'll highlight some of the most important settings that you can use to get
+the best performance in your environment.
+
+## Puppet Server and JRuby
+
+Before you begin tuning your configuration, it's helpful to have a little bit
+of context on how Puppet Server uses JRuby to handle incoming HTTP requests from
+your Puppet agents.
+
+When Puppet Server starts up, it creates a pool of JRuby interpreters to use
+as workers when we need to execute some of the Puppet Ruby code.  You can think
```

Comment:
I think it's preferable to avoid the "we" working here ... something better would be "When Puppet Server starts up, it creates a pool of JRuby interpreters to use as workers when __it needs__ to execute some of the Puppet Ruby code."

@nfagerlund you're welcome :)